### ISSUE TITLE
Example ps.error not working in python

### ISSUE THREAD

**Issue by JPLost** [nmwsharp/polyscope#277]
On page https://polyscope.run/py/features/messages/ the code example on last line does not work. Does not print error message to Polyscope window. Python 3.9 and PyCharm 2024.1.1 (Community Edition)
```py
# Generating an error.
# The UI will block and show this error immediately. After 
# the error is dismissed, the call will return.
ps.error("Resistance is futile.");

**Comment by nmwsharp** [comment]
You're right! This was a bug in the core C++ library. For some reason the `error()` function lacked the code to create the onscreen dialog, even though it was totally meant to. Perhaps I accidentally removed it at some point.

Fixed in [edb6a36](https://github.com/nmwsharp/polyscope/commit/edb6a36399d21a24b05c0d15355887c0748d39a9). The fix will be propagated to the published Python library in the next release.

Thanks for the bug report!
